packages/watt-pprof-capture/index.js

```diff
@@ -72,8 +82,39 @@ function getLastProfile () {
   return latestProfile.encode()
 }
 
+async function generateProfile (options = {}) {
```

Comment:
If we are already profiling this should throw

Reply:
done